python/src/ResourceMap.i

```diff
@@ -16,7 +16,7 @@ namespace OT {
 
 %extend ResourceMap {
 
-UnsignedInteger __len__() const
+static UnsignedInteger __len__() const
```

Comment:
ResourceMap.i:22: Error: Static function __len__() const cannot have a qualifier.

Reply:
Maybe it only fails with swig master here?

Reply:
Indeed it does not make sense to have a const qualifier for static methods, I'll drop it.